### DISCUSSION TITLE
Fail to install superset in ibm cloud IKS (ibm cloud k8s service)

### DISCUSSION THREAD

**Discussion by graceguo111** [apache/superset#26780] [Q&A / Help]
I want install superset follow this guide in ibm cloud IKS (ibm cloud k8s service): https://superset.apache.org/docs/installation/running-on-kubernetes#

but get Init:ImagePullBackOff error when pulling superset image:

Failed to pull image "apache/superset:dockerize": rpc error: code = NotFound desc = failed to pull and unpack image "docker.io/apache/superset:dockerize": no match for platform in manifest: not found

for ibm cloud IKS, I install ibm cloud cli and kubectl in my local window PC, so I manage k8s cluster from my local PC.

**Comment by magalibarreaucombeau** [comment]
I have the same issue since yesterday morning but it was working 2 days ago. I am still investigating it.

**Comment by graceguo111** [comment]
The issue is gone, I did nothing.

From: Magali Barreau-Combeau ***@***.***>
Sent: Thursday, January 25, 2024 3:46 PM
To: apache/superset ***@***.***>
Cc: Yuan Xu AG Guo ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [apache/superset] Fail to install superset in ibm cloud IKS (ibm cloud k8s service) (Discussion #26780)

I have the same issue since yesterday morning but it was working 2 days ago. I am still investigating it. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread. Message
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
    Report Suspicious  <https://us-phishalarm-ewt.proofpoint.com/EWT/v1/PjiDSg!12-r6rRVIbM5Uk0T0ikN5OcGWIVMHNMznDlu93kP_J-PYjF1d4gBffmnSAba32KWyjRFUSPj3QqbMBatnWa-Krudk3WCUuAQ0MesnbR-t6VD3fqqUTehyCMG-l0$>   ‌
ZjQcmQRYFpfptBannerEnd

I have the same issue since yesterday morning but it was working 2 days ago. I am still investigating it.

**Comment by tsschuel** [comment]
I get the same problem on AWS btw...

**Comment by magalibarreaucombeau** [comment]
hello,
here is the solution I applied: 

helm setup of 0.12.0 fails because the dockize image is not found
This has been reported : https://github.com/apache/superset/issues/26788
the workaround is to use 
'5040044ee638e5e41d572deaa2f8547d02d05449-dockerize-linux-amd64-3.9-slim-bookworm' image instead
dockerize image

I modified docker image used by added the following into the values.yaml file :
`initImage:
  repository: apache/superset
  tag: 5040044ee638e5e41d572deaa2f8547d02d05449-dockerize-linux-amd64-3.9-slim-bookworm
`

**Reply by kraftaa** [reply]
they just pushed an updated image, it's working 
```
dockerize
docker pull apache/superset:dockerize
Last pushed 7 minutes ago by [asfjenkins]
```

though 1 pod worked, another is still `Init:ErrImagePull`

**Comment by tsschuel** [comment]
Hi, 

That seemed to have helped a bit but still the worker seems to have the same issue:


```
Events:
  Type     Reason     Age                From               Message
   ----     ------     ----               ----               -------
  Normal   Scheduled  97s                default-scheduler  Successfully assigned superset/superset-worker-6956d6bdf8-4kwcj to xxx.compute.internal
  Normal   BackOff    23s (x4 over 93s)  kubelet            Back-off pulling image "apache/superset:dockerize"
  Warning  Failed     23s (x4 over 93s)  kubelet            Error: ImagePullBackOff
  Normal   Pulling    9s (x4 over 96s)   kubelet            Pulling image "apache/superset:dockerize"
  Warning  Failed     7s (x4 over 93s)   kubelet            Failed to pull image "apache/superset:dockerize": rpc error: code = NotFound desc = failed to pull and unpack image "docker.io/apache/superset:dockerize": no match for platform in manifest: not found
  Warning  Failed     7s (x4 over 93s)   kubelet            Error: ErrImagePull
```

Anything I am missing?

**Reply by kraftaa** [reply]
I'm waiting for them merging the fix: https://github.com/apache/superset/pull/26787
